doc/Learn/Maui/MauiOverview.md

```diff
@@ -49,6 +51,28 @@ dotnet new unoapp -preset blank -maui -o SimpleMauiApp
 
 .NET MAUI Embedding can be added to any existing Uno application via the following steps. The .NET MAUI Embedding feature is only supported in Uno applications that target iOS, Android, MacCatalyst, and Windows, as shown in the Platforms folder in the following solution structure. These instructions are appropriate for an Uno application targeting .NET 7.
```

Comment:
```suggestion
.NET MAUI Embedding can be added to any existing Uno application via the following steps. The .NET MAUI Embedding feature is only supported in Uno applications that target iOS, Android, MacCatalyst, and Windows, as shown in the Platforms folder in the following solution structure. These instructions are appropriate for an Uno application targeting .NET 8.
```

I imagine we want to change for .NET 8 here

Reply:
Maybe we remove that line, gonna be annoying to update everytime we move to the new .NET version

Reply:
Line removed.